torchao/csrc/cuda/tensor_core_tiled_layout/tensor_core_tiled_layout.cu

```diff
@@ -15,18 +19,18 @@ constexpr __host__ __device__ auto divUp(U a, V b) -> decltype(a + b) {
 }
 constexpr int32_t kWarpSize = 32;
 
-//Simple data structure to represent 4 pairs of bfloat16s, used for vectorized dequantization
```

Comment:
UGH IGNORE the random VSCode formats--Idk how autoformat is turned on in this repo for me + will figure it out later, skip this line til 170!!!